docs/runtime-contract.md

```diff
@@ -484,16 +484,16 @@ From the OCI spec:
 > article in the kernel documentation has more information about mount
 > propagation.
 
-This option should only be set by the operator or platform provider, and MUST
-NOT be configurable by the developer. As mount propagation may be part of the
-platform security hardening, operators MAY tune this over time as the threat
+This option SHOULD only be set by the operator or platform provider, and MUST
```

Comment:
```suggestion
This option MAY be set by the operator or platform provider, and MUST
```

The second part of the sentence clarifies the 'only' part so that is unnecessary. We don't provide any real guidance or requirement here so SHOULD seems to be too strong of a word choice.

Reply:
ok done